MotorControl/MotorControl/Resources/MotorControl.strings

```diff
@@ -36,9 +36,9 @@
 
 "TAP_COUNT_LABEL" = "TAP COUNT";
 
-"ACTIVE_STEP_UNIT_LABEL" = "SECONDS";
+"ACTIVE_STEP_UNIT_LABEL" = "Seconds";
 
-"ACTIVE_STEP_UNIT_LABEL_SINGULAR" = "SECOND";
+"ACTIVE_STEP_UNIT_LABEL_SINGULAR" = "Second";
```

Comment:
For consistency, since the design team wants to change this to capitalized, should also change "TAP COUNT" above.

Reply:
Done 👍

Reply:
thanks